# error=> x is not defined


try:
  x=10 # comment this and run
  print(x)
except:
  print("An exception occurred")